versions.gradle

```diff
@@ -7,9 +7,9 @@
 
 // casual-java group and version
 group = 'se.laz.casual'
-version = '3.2.23'
+version = '3.2.24'
```

Comment:
We could bump it, it does not really break the api since we're only adding to it.
However, we should start getting in the habit of bumping minor for well, minor non breaking changes :)